app/models/user.rb

```diff
@@ -0,0 +1,41 @@
+class User < ApplicationRecord
+  has_many :votes
+  validates :username, presence: true, uniqueness: true
+
+
+  # create a vote object with current user's id and the work id passed in
+  def vote(work_id)
+    vote = Vote.where(user_id: self.id, work_id: work_id)
+    return vote
```

Comment:
Your comments say that this "create"s a vote object, but it calls `Vote.where`... it doesn't create a Vote, it just finds any existing votes, and returns it as an array